components/expand-collapse/expand-collapse-content.js

```diff
@@ -44,18 +46,34 @@ class ExpandCollapseContent extends LitElement {
 
 			.d2l-expand-collapse-content-container {
 				display: none;
+				opacity: 0;
 				overflow: hidden;
-				transition: height 400ms cubic-bezier(0, 0.7, 0.5, 1);
+				transition:
+					height var(--d2l-expand-collapse-content-transition-duration) var(--d2l-expand-collapse-content-transition-function),
+					opacity var(--d2l-expand-collapse-content-transition-duration) var(--d2l-expand-collapse-content-transition-function);
 			}
 
 			.d2l-expand-collapse-content-container:not([data-state="collapsed"]) {
 				display: block;
 			}
 
+			.d2l-expand-collapse-content-container[data-state="collapsed"] {
+				opacity: 0;
+			}
+
+			.d2l-expand-collapse-content-container[data-state="collapsing"] {
+				opacity: 0;
+			}
+
 			.d2l-expand-collapse-content-container[data-state="expanded"] {
+				opacity: 1;
 				overflow: visible;
 			}
 
+			.d2l-expand-collapse-content-container[data-state="expanding"] {
+				opacity: 1;
+			}
```

Comment:
I'm not sure which way I prefer in this case... I probably prefer grouping by property in this case but... 🤷 
```suggestion
			.d2l-expand-collapse-content-container[data-state="expanded"] {
				overflow: visible;
			}

			.d2l-expand-collapse-content-container[data-state="expanded"],
			.d2l-expand-collapse-content-container[data-state="expanding"] {
				opacity: 1;
			}
```